openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx

```diff
@@ -457,139 +812,174 @@ const PipelineDetails = ({
               ? onTierUpdate
               : undefined
           }
-          version={version}
+          version={version + ''}
```

Comment:
is it intentional? if the purpose is to convert to string can we try `${version}`

Reply:
yes intentional and required